tibiawikisql/utils.py

```diff
@@ -297,7 +300,8 @@ def _parse_loot_entry(entry):
             key = subarg[0].strip()
             value = subarg[1].strip()
         entry[key] = value
-    return entry
+    if "item" in entry:
+        return entry
```

Comment:
If function might return a value, add an explicit `return None`.

The function that calls this should then handle None entries and filter them out.

Reply:
I can add the explicit `return None`.